### ISSUE TITLE
FNIS not working correctly+temp resolution

### ISSUE THREAD

**Issue by Gernash** [TanninOne/modorganizer#878]
IF you install FNIS+PCEA2+Creature Pack GenerateFNISforUSERS Crashes

Workaround:
I have an win10 dual Hdd system and My MO2 is on the 2nd Hdd. 
I had to install MO2 and start it once
then move the Mod Organizer folder from AppData to my 2nd Hdd
then create a symbolic link. mklink /J C:\Users{User}\AppData\Local\ModOrganizer J:/{My new folder}
This resolves overwrite folder issues
Recommend deleting all nxmhandler.ini and ModOrganizer.ini and allow MO2 to recreate them also run nxmhandler.exe in MO2 root folder to generate the nxmhandlers.ini files

FNIS+PCEA2+Creature Pack+FNIS Spells have to be merged into a single Mod.
GenerateFNISforUSERS works and finds XP32 Maximum Skeleton Extended (Does not have to be merged)

**Comment by michaelrw10** [comment]
Why are you trying to run skyrim on MO2 when there is absolutely no need.. 1.3.11 is perfectly stable for skyrim and there is a lot more support for it from the community (i.e. STEP) ... i feel like getting mo2 stable on fallout4 is and should be the primary focus. only after that has happened should we be trying to add more games. just my opinion.

**Comment by Gernash** [comment]
My normal play game is on 1.3.11 I use MO2 for fallout 4 (about 124 Mods stable no CDT).
MO2 is for people testing with an issue tracker for problems I have posted an issue (I've had none in fallout 4 other than the folder move one and eagerly waiting for the next release with the Commits on Jun 24, 2016 hence i posted a work around till it's out.)

Sharing is caring with open source IMO.
